facia/test/FaciaControllerTest.scala

```diff
@@ -143,13 +143,6 @@ import scala.concurrent.Await
     faciaController.alternativeEndpoints("uk/business/stock-markets") should be (List("business", "uk"))
   }
 
-  it should "render fronts with content that has been pre-fetched from facia-press" in {
-    // make sure you switch on facia inline-embeds switch before you press to make this pass
-    val request = FakeRequest("GET", "/inline-embeds").from(Uk)
-    val future = faciaController.renderFront("inline-embeds")(request)
-    contentAsString(future) should include ("""<div class="keep-it-in-the-ground__wrapper""")
-  }
-
```

Comment:
This test is super fragile. Any change of this dev only fronts or the embed code itself makes it fails.
Also it is testing a Facia-press feature and therefore doesn't belong in `FaciaControllerTest`
I am open to suggestion for testing this feature in a solid way